src/Tuura/Concept/FSM/Translation.hs

```diff
@@ -158,9 +193,12 @@ fullListm :: ([TransitionX a], Transition a) -> [TransitionX a]
 fullListm (l,t) = (toTransitionX t):l
 
 -- Given [([a], b)], remove all b from a
-removeDupes :: Eq a => [([Transition a], Transition a)] -> [([Transition a], Transition a)]
+removeDupes :: Eq a => [([Transition a], Transition a)] ->
+                       [([Transition a], Transition a)]
 -- (filter ((/= ((signal . snd) x)) . signal) (fst x), snd x)
-removeDupes = map (ap ((,) . ap (filter . (. signal) . (/=) . signal . snd) fst) snd)
+removeDupes = map (ap
+              ((,) . ap (filter . (. signal) . (/=) . signal . snd) fst)
```

Comment:
This one's my fault. I didn't figure how to implement the function in non-point-free form at the time :p @jrbeaumont Let me know if you need help reworking it.